src/main/java/org/spongepowered/common/config/category/CommandsCategory.java

```diff
@@ -41,6 +41,9 @@
             + "changes on the all worlds.")
     private Map<String, Boolean> multiWorldCommandPatches = new HashMap<>();
 
+    @Setting(value = "force-permission-handler", comment = "Forces the Sponge permission handler to be used rather than another mod's permission handler")
+    private boolean forcePermissionHandler = true;
```

Comment:
Why is this in the commands category? Permissions have nothing to do with commands.